web/themes/custom/cloudy/src/js/back-to-top.js

```diff
@@ -0,0 +1,27 @@
+import $ from 'jquery';
+import Drupal from 'Drupal';
+
+Drupal.behaviors.cloudy_back_to_top = {
+  attach: function(context, settings) {
+    var viewHeight = $(window).height();
+    var showHeight = viewHeight * 1.5;
+    var isAttached = false;
+
+    $(window).once('backToTopShowButtonHandler').on('scroll', function() {
+      var scrollPos = $(document).scrollTop();
+      if (scrollPos > showHeight && !isAttached) {
+        $('#block-cloudy-content').append('<div id="back-to-top" class="btn btn-dark"><a href="#main-content">Back to top</a></div>');
```

Comment:
Selector context and Drupal translate on the button text have been added.

I'm going to leave the tab order alone for now. It makes sense to me semantically that it's attached to the bottom of the content area, not the footer. If a user has navigated other pages already, it's more work for them to have to tab through the footer, which they've already seen on all other pages.

I don't think the sr-only text is necessary since screen readers would read the button.